### ISSUE TITLE
I get a TypeError in check_header_validity

### ISSUE THREAD

**Issue by HeRo002** [zbears/pyvona#31]
Hi

I am trying out your fascinating Pyvona package with this simple test: 

> import pyvona
> 
> v = pyvona.create_voice("BvtHy743Ds1gPCq", "BvtHy743Ds1gPCq")
> v.speak("Hello World")

But I get the TypeError below:

NB!: When I signed up for Ivona it did not ask me to make 2 passwords, so I am using the same password as both ‘Access Key’ and ‘Secret Key’. 

(PS: I would love to make this message more readable, but I can't see any appropriate buttons here above the editor.)

----Start of error message-----
"C:\Program Files (x86)\Python35-32\python.exe" D:/DataDoc/PycharmProjects/Pyvona-test.py
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\utils.py", line 796, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:/DataDoc/PycharmProjects/Pyvona-test.py", line 4, in <module>
    v.speak("Hello World")
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\pyvona.py", line 148, in speak
    self.fetch_voice_fp(text_to_speak, f)
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\pyvona.py", line 123, in fetch_voice_fp
    self._generate_payload(text_to_speak), self._region, self._host)
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\pyvona.py", line 254, in _send_amazon_auth_packet_v4
    return self._session.post(endpoint, data=request_parameters, headers=headers)
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\sessions.py", line 522, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\sessions.py", line 461, in request
    prep = self.prepare_request(req)
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\sessions.py", line 394, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\models.py", line 295, in prepare
    self.prepare_headers(headers)
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\models.py", line 409, in prepare_headers
    check_header_validity(header)
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\utils.py", line 800, in check_header_validity
    "not %s" % (value, type(value)))
requests.exceptions.InvalidHeader: Header value 241 must be of type str or bytes, not <class 'int'>

Process finished with exit code 1
----End of error message-----

I am using Python 3.5.2 32-bit on PyCharm 2016.2.3 on Windows 10 64-bit. 

Thank you for any help with this! 

PS: I also reported this on http://zacharybears.com/pyvona/#comment-8421

Henrik R.

**Comment by garethellis0** [comment]
Hey @HeRo002 , the secret and access keys are not the passwords to your Ivona account (and if the above is your actual password, you might want to go change it now). Rather, they are credentials you generate under `"Speech Cloud"  -> "Credentials"` on the Ivona site. Once you've created the credentials, they may take a while to activate, I've had to wait up to 8-24 hours for them to become active. Also, for future reference, you can use the ``` tags on the line before and after your stack trace or code to format it, like so:

```
----Start of error message-----
"C:\Program Files (x86)\Python35-32\python.exe" D:/DataDoc/PycharmProjects/Pyvona-test.py
Traceback (most recent call last):
File "C:\Program Files (x86)\Python35-32\lib\site-packages\requests\utils.py", line 796, in check_header_validity
if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:
requests.exceptions.InvalidHeader: Header value 241 must be of type str or bytes, not

Process finished with exit code 1
----End of error message-----
```

Also, you can just highlight your code/stack trace and hit the "<>" button in the top bar to accomplish the same thing. I'm going to close this issue, but feel free to open it again if you've got more problems.

**Comment by HeRo002** [comment]
Thank you for your help!

I logged in to Ivona and clicked on Speech Cloud -> Sign-up. But when I discovered how much information Ivona wants on me and my project I decided to switch to gtts - the Google text-to-speech: 
https://github.com/pndurette/gTTS 

(So the rest of this message is not directly related to Pyvona!)

So far this works:

```
from gtts import gTTS

audio_file = "hello.mp3"
tts = gTTS(text="Hello World!", lang="en")
tts.save(audio_file)
```

The problem is how to play the sound. I tried 4 different solutions. Only the playsound package worked: 

```
from playsound import playsound
playsound('/path/to/a/sound/file/you/want/to/play.mp3')
```

But only after making the changes to playsound.py described here: https://github.com/TaylorSMarks/playsound/pull/3/files 

But it would be more elegant not having to write the audio-data to a file and then play that file...!

And on http://stackoverflow.com/questions/30612298/text-to-speech-tts-module-that-works-under-python-3 someone seems to suggest that on OS X you don't even need to call gTTS (or Pyvona)??: 

`If you're going to use subprocess on OS X, why not just subprocess.call(["say", "Hello World!"])`

Any comments are welcome. I am sorry this is not directly related to Pyvona!

Best regards,
Henrik R.
